app/models/miq_server/rhn_mirror.rb

```diff
@@ -1,169 +0,0 @@
-require 'linux_admin'
-require 'miq_apache'
-
-module MiqServer::RhnMirror
-  extend ActiveSupport::Concern
-  APACHE_MIRROR_CONF_FILE = "/etc/httpd/conf.d/manageiq-https-mirror.conf"
-  YUM_MIRROR_CONF_FILE    = "/etc/yum.repos.d/manageiq-mirror.repo"
```

Comment:
Are these files present anywhere that they need to be deleted?  How about on upgrade?